Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -46,16 +47,21 @@
 import org.kitodo.production.metadata.copier.DataCopier;
 import org.kitodo.production.model.Subfolder;
 import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.data.ProcessService;
 import org.kitodo.production.services.file.FileService;
 import org.kitodo.production.services.workflow.WorkflowControllerService;
 
 public class ExportDms extends ExportMets {
     private static final Logger logger = LogManager.getLogger(ExportDms.class);
-    private boolean exportWithImages = true;
-    private final FileService fileService = ServiceManager.getFileService();
+    private static final String COMPLETED = "100000000000";
```

Comment:
Ignore it. As the state of task is calculated on the fly (which could result in bad / slow performance) instead of using the stored value inside the process (value is updated every time a state of task is changed) the here used value is correct.